odo/numpy_dtype.py

```diff
@@ -23,6 +25,9 @@ def unit_to_dtype(ds):
     if isinstance(ds, DataShape):
         ds = ds.measure
     if isinstance(ds, Option) and isscalar(ds) and isnumeric(ds):
+        with ignoring(NameError):
+            if isinstance(ds.ty, Decimal):
+                return unit_to_dtype(str(ds.ty.to_numpy_dtype()).replace('int', 'float'))
```

Comment:
@octophat , cool can you remove the `ignoring(NameError)` block and we can get this in?